Content.Server/GameObjects/Components/Items/rcdComponent.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using Content.Server.GameObjects.Components.Stack;
+using Content.Server.GameObjects.EntitySystems;
+using Content.Server.Interfaces;
+using Content.Shared.Maps;
+using Microsoft.EntityFrameworkCore.Internal;
```

Comment:
Looks like a mistaken import here, probably from your editor trying to fix a missing name?